blockchain/projects.go

```diff
@@ -369,3 +391,26 @@ func (p *Projects) filterMissingExecutions(
 
 	return executions, nil
 }
+
```

Comment:
this get account function would be better to be grouped above with the exported func

Reply:
Done.